habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1667,7 +1668,7 @@ class AgentConfig(HabitatBaseConfig):
     max_climb: float = 0.2
     max_slope: float = 45.0
     grasp_managers: int = 1
-    sim_sensors: Dict[str, SimulatorSensorConfig] = field(default_factory=dict)
```

Comment:
@aclegg3 

This field has a bad smell.
When refactoring sensors, we should revisit how users define their sensors. If you look just above the code here, you'll notice that framework-level subclasses were created for application-level code.
We should aim to use composition rather than inheritance to define sensors.